### ISSUE TITLE
About requirements.txt

### ISSUE THREAD

**Issue by joysieun** [OpenImagingLab/TimeLens-XL#5]
I noticed the following entry in the requirements.txt file:
file:///croot/aiobotocore_1714464387507/work

It seems to reference a local file path, and I’m encountering errors during installation because this file is missing. Is there an alternative way to handle this?

Additionally, if this file is necessary, could you kindly send it to my email at sseenn677@gmail.com?

**Comment by pengbo7** [comment]
Hi, I have the same installation problem when configuring the environment.May I ask how you solved it?

**Comment by joysieun** [comment]
This is my requirements.txt file. 

absl-py==2.1.0
accelerate==1.3.0
addict==2.4.0
asttokens==3.0.0
bitsandbytes==0.45.0
Brotli @ file:///home/conda/feedstock_root/build_artifacts/brotli-split_1725267488082/work
certifi==2025.1.31
cffi @ file:///home/conda/feedstock_root/build_artifacts/cffi_1725560520483/work
cfgv==3.4.0
charset-normalizer @ file:///home/conda/feedstock_root/build_artifacts/charset-normalizer_1735929714516/work
click==8.1.8
colorama @ file:///home/conda/feedstock_root/build_artifacts/colorama_1733218098505/work
contourpy==1.3.1
cycler==0.12.1
distlib==0.3.9
easydict==1.13
einops==0.8.0
exceptiongroup==1.2.2
executing==2.1.0
facexlib==0.3.0
filelock @ file:///home/conda/feedstock_root/build_artifacts/filelock_1737517818712/work
filterpy==1.4.5
flow-vis==0.1
fonttools==4.55.3
fsspec @ file:///home/conda/feedstock_root/build_artifacts/fsspec_1738501244951/work
ftfy==6.3.1
future==1.0.0
gmpy2 @ file:///home/conda/feedstock_root/build_artifacts/gmpy2_1733462536562/work
grpcio==1.68.1
h2 @ file:///home/conda/feedstock_root/build_artifacts/h2_1738578511449/work
hpack @ file:///home/conda/feedstock_root/build_artifacts/hpack_1737618293087/work
huggingface_hub @ file:///home/conda/feedstock_root/build_artifacts/huggingface_hub_1738349326790/work
hyperframe @ file:///home/conda/feedstock_root/build_artifacts/hyperframe_1737618333194/work
icecream==2.1.3
identify==2.6.3
idna==3.10
imageio==2.36.1
iniconfig==2.0.0
Jinja2==3.1.5
kiwisolver==1.4.7
lazy_loader==0.4
llvmlite==0.43.0
lmdb==1.5.1
Markdown==3.7
MarkupSafe==3.0.2
matplotlib==3.10.0
mpmath==1.3.0
natsort==8.4.0
networkx==3.4.2
nodeenv==1.9.1
numba==0.60.0
numpy==2.0.0
nvidia-cublas-cu11==11.11.3.6
nvidia-cublas-cu12==12.4.5.8
nvidia-cuda-cupti-cu11==11.8.87
nvidia-cuda-cupti-cu12==12.4.127
nvidia-cuda-nvrtc-cu11==11.8.89
nvidia-cuda-nvrtc-cu12==12.4.127
nvidia-cuda-runtime-cu11==11.8.89
nvidia-cuda-runtime-cu12==12.4.127
nvidia-cudnn-cu11==9.1.0.70
nvidia-cudnn-cu12==9.1.0.70
nvidia-cufft-cu11==10.9.0.58
nvidia-cufft-cu12==11.2.1.3
nvidia-curand-cu11==10.3.0.86
nvidia-curand-cu12==10.3.5.147
nvidia-cusolver-cu11==11.4.1.48
nvidia-cusolver-cu12==11.6.1.9
nvidia-cusparse-cu11==11.7.5.86
nvidia-cusparse-cu12==12.3.1.170
nvidia-cusparselt-cu12==0.6.2
nvidia-nccl-cu11==2.20.5
nvidia-nccl-cu12==2.21.5
nvidia-nvjitlink-cu12==12.4.127
nvidia-nvtx-cu11==11.8.86
nvidia-nvtx-cu12==12.4.127
openai-clip==1.0.1
opencv-python==4.10.0.84
opencv-python-headless==4.10.0.84
packaging==24.2
pandas==2.2.3
pillow==11.0.0
platformdirs==4.3.6
pluggy==1.5.0
pre_commit==4.0.1
protobuf==5.29.2
psutil==6.1.1
pybind11 @ file:///D:/bld/pybind11-split_1730237329882/work
pybind11_global @ file:///home/conda/feedstock_root/build_artifacts/pybind11-split_1730237328107/work
pycparser @ file:///home/conda/feedstock_root/build_artifacts/bld/rattler-build_pycparser_1733195786/work
Pygments==2.18.0
pyiqa==0.1.13
pyparsing==3.2.0
PySocks @ file:///home/conda/feedstock_root/build_artifacts/pysocks_1733217236728/work
pytest==8.3.4
python-dateutil==2.9.0.post0
pytorch-model-summary==0.1.2
pytz==2024.2
PyYAML==6.0.2
regex==2024.11.6
requests==2.32.3
safetensors @ file:///home/conda/feedstock_root/build_artifacts/safetensors_1736382933512/work
scikit-image==0.25.0
scipy==1.14.1
sentencepiece==0.2.0
six==1.17.0
sympy==1.13.1
tensorboard==2.18.0
tensorboard-data-server==0.7.2
thop==0.1.1.post2209072238
tifffile==2024.12.12
timm==1.0.12
tokenizers==0.15.2
tomli==2.2.1
torch==2.4.1+cu118
torchaudio==2.4.1+cu118
torchinfo==1.8.0
torchsummary==1.5.1
torchvision==0.19.1+cu118
tqdm==4.67.1
transformers==4.37.2
triton==3.0.0
typing_extensions==4.12.2
tzdata==2024.2
urllib3==2.3.0
virtualenv==20.28.0
wcwidth==0.2.13
Werkzeug==3.1.3
yapf==0.43.0
zstandard==0.23.0

**Comment by pengbo7** [comment]
Thank you very much, I have trained according to the author's profile but the metrics are not as good as the pre-training weights given by the author, I wonder how it is in your case?
